earth2mip/initial_conditions/gfs.py

```diff
@@ -419,10 +426,11 @@ def _get_gfs_name_dict() -> Dict[str, str]:
 
 
 class DataSource(base.DataSource):
-    def __init__(self, channels: List[str]) -> None:
+    def __init__(self, channels: List[str], download_ic=True) -> None:
```

Comment:
Recommend adding type hints, so `download_ic: bool = True`

Reply:
good suggestion, adding in.